Update Documentation / WatchGuide including images.

kskandispersonal opened the pull request.

jamorham commented on app/src/main/AndroidManifest.xml:
```diff
@@ -54,7 +54,7 @@
         <meta-data
             android:name="AA_MODELS"
-            android:value="com.eveningoutpost.dexdrip.Models.ActiveBgAlert,com.eveningoutpost.dexdrip.Models.ActiveBluetoothDevice,com.eveningoutpost.dexdrip.Models.AlertType,com.eveningoutpost.dexdrip.Models.BgReading,com.eveningoutpost.dexdrip.Models.BgReading,com.eveningoutpost.dexdrip.Models.Calibration,com.eveningoutpost.dexdrip.Models.Calibration,com.eveningoutpost.dexdrip.Models.Calibratio ...
+            android:value="com.eveningoutpost.dexdrip.Models.ActiveBgAlert,com.eveningoutpost.dexdrip.Models.ActiveBluetoothDevice,com.eveningoutpost.dexdrip.Models.AlertType,com.eveningoutpost.dexdrip.Models.BgReading,com.eveningoutpost.dexdrip.Models.BgReading,com.eveningoutpost.dexdrip.Models.Calibration,com.eveningoutpost.dexdrip.Models.Calibration,com.eveningoutpost.dexdrip.Models.Calibratio ...
```
This has pebble movement in it twice. I assume modification of the app manifest was accidental.

kskandispersonal commented:
Added support for Step Counter.  It is using the PebbleMovement table, as well as the use_pebble_health preference.  This way, nothing in BgGraphBuilder needs to be changed (at least I don't think changes are required!).  Step Counter will display on the watch as well if the user enables the "showSteps" pref XDrip Prefs.  PebbleMovement data is only sent back to the phone if use_pebble_health is enabled on the phone.

I also added fixes for gitter issue:
Chrisalias @Chrisalias 
16:30
@kskandispersonal If I open XDrip Prefs (1st page) and just leave it open, my SW3 goes into ambient mode displaying the watch face, ie I don't see XDrip Prefs in ambient mode ever. It takes about 20 seconds.

I used Adrian's suggested resolution:
@Override
    protected void onPause(){
        super.onPause();
        finish();

I also added imports for UserError.Log to ListenerService.  It was defaulting to import android.util.Log;  I'm not sure if this is related to the duplicate log entries that Jim E @TecMunky  on gitter is seeing.  I don't think it is, because he is seeing dup entries in G5CollectionService as well which already uses UserError.Log. For example, his screenshot shows dups on the following log entry:
    private synchronized void doDisconnectMessage(BluetoothGatt gatt, BluetoothGattCharacteristic characteristic) {
           Log.d(TAG, "doDisconnectMessage() start");

But no dups on the following entry:
    private synchronized void processNewTransmitterData(int raw_data , int filtered_data,int sensor_battery_level, long captureTime) {
        Log.d(TAG,"Dex raw_data "+ Double.toString(transmitterData.raw_data));//KS

I'm not sure how consistent this is, or whether the actual log entries are stored in his UserError table.  I will do a bit more investigation with my UserError table...

I think overall the Step Counter is ready to merge but I will test tonight to be sure counters reset tomorrow!
